content/departments/product-engineering/engineering/cloud/devops/managed/restore_process.md

```diff
@@ -0,0 +1,39 @@
+# Restore a managed instance when it is completely gone
+
+<span class="badge badge-note">SOC2/CI-110</span>
+
+## Assess what is deleted
```

Comment:
@michaellzc for v1.1 we also need to add CloudSQL restore process - should I extend your PR? Or do it in separate?

Also for [SOC2-111](https://github.com/sourcegraph/security-issues/issues/247) I need to perform all actions from this page, write date-time and prove we do it annually - I will do it in google doc and ask you to review, ok?

Reply:
Hey @filiphaftek 

there's a separate thread where I was trying to get more clarify on what's required https://sourcegraph.slack.com/archives/C02Q9G9A59S/p1653334149100629

but yeah.

I am updating this PR to include process to restore Cloud SQL for MI v1.1 as well.

> Also for https://github.com/sourcegraph/security-issues/issues/247 I need to perform all actions from this page, write date-time and prove we do it annually - I will do it in google doc and ask you to review, ok?

yes please, thank you!